modules/cloudfront_distribution/tf_module/main.tf

```diff
@@ -7,9 +8,68 @@ data "aws_s3_bucket" "logging_bucket" {
   bucket = var.s3_log_bucket_name
 }
 
+locals {
+  lb_distribution_count = var.eks_load_balancer_enabled == true ? 1 : 0
+  s3_distribution_count = var.s3_load_balancer_enabled == true ? 1 : 0
+}
+
+resource "aws_cloudfront_distribution" "lb_distribution" {
```

Comment:
@NitinAgg thoughts on having a separate resource? I know I dug my own grave by first calling it `s3_distribution`, but we ought to just have a single resource so that we can easily update to support going to both s3 AND eks depending on path.

Reply:
Agree. Let's try to fit this in the existing module. We can change the resource name. No one is using this in prod yet

Reply:
@NitinAgg @juandiegopalomino @bigbitbus have a look at these changes.
For references, you can also have a look at these Cloudfront Distributions (Runx-Tmp AWS Account) as well:
* E2TICDHTZE41Y0
* E11KTM0ZEFNT3A